src/components/notification-badge/NotificationBadge.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+
+import { Box } from '../box'
+import { Flex } from '../flex'
+
+import { styled } from '~/stitches'
+
+const StyledWrapper = styled(Box, {
+  position: 'relative'
+})
+
+const StyledBadge = styled(Flex, {
+  fontFamily: '$body',
+  fontSize: '$sm',
+  flexDirection: 'column',
+  justifyContent: 'center',
+  position: 'absolute',
+  top: '0',
+  right: '0',
+  transform: 'translate(calc($sizes$2 / 2), -50%)',
+  borderRadius: '$round',
+  p: '$2',
+  height: '$2',
+  minWidth: '$2',
+  textAlign: 'center',
+  color: '#fff',
+  bg: '$primary'
+})
+
+type NotificationBadgeProps = React.ComponentProps<typeof StyledBadge> & {
+  value: number | string
+}
+
+export const NotificationBadge: React.FC<NotificationBadgeProps> = ({
+  value,
+  children
+}) => (
+  <StyledWrapper>
+    {value ? <StyledBadge role="status">{value}</StyledBadge> : ''}
```

Comment:
```suggestion
    {!!value && <StyledBadge role="status">{value}</StyledBadge>}
```

Could we convert the value to a boolean for this conditional? It should prevent the "render a 0" problem and might be a better solution than rendering an empty string 👍

Reply:
yeah that makes more sense. I always forget about `!!`. will fix now :)